test/unit/bucket/test_bucket.py

```diff
@@ -1568,8 +1568,10 @@ def test_upload_file_one_fatal_error(self):
             self.bucket.upload_bytes(data, 'file1')
 
     def test_upload_file_too_many_retryable_errors(self):
-        self.simulator.set_upload_errors([CanRetry(True)] * 6)
+        self.simulator.set_upload_errors([CanRetry(True)] * 3)
         data = b'hello world'
+        self.api.services.upload_manager.set_retry_time(5)
+        self.api.services.upload_manager.RETRY_INTERVAL_TIME = 2
         with self.assertRaises(MaxRetriesExceeded):
             self.bucket.upload_bytes(data, 'file1')
```

Comment:
this test doesn't actually test things, only pretends to do so, polluting the output of code coverage analyzer with unchecked passthroughts